### ISSUE TITLE
Distutils removal warning for Python 3.12

### ISSUE THREAD

**Issue by greatmastermario** [overviewer/Minecraft-Overviewer#2013]
There is currently a deprecation warning when running setup.py. It says that distutils is being removed from Python starting in version 3.12. This will break C library builds if this version is used. We need to research an alternative to this library for C builds.

**Comment by fletchowns** [comment]
With the release if Debian Trixie, the default python version is now 3.13. Workaround would be to install an older version of Python to use with Minecraft Overviewer.

**Comment by CounterPillow** [comment]
no, the workaround is to stop using this because it's unmaintained

**Comment by fletchowns** [comment]
Ah! I missed the announcement. Thank you for all the hard work on this project! It was much appreciated :)

**Comment by greatmastermario** [comment]
I took a look into what it would take to stop using distutils, and it's honestly not a lot. Using the latest Python (3.13.7) I was able to get all the imports fixed. I have not yet opened a PR because of a build failure, but this is what I have so far: https://github.com/overviewer/Minecraft-Overviewer/compare/master...greatmastermario:Minecraft-Overviewer:Issue2013-DistUtils-Removal?expand=1

Regarding that build failure, I see the following error when running the setup.py build:
`running build_ext
building 'overviewer_core.c_overviewer' extension
"C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.44.35207\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -I. -IC:\Users\maste\IdeaProjects\Minecraft-Overviewer\.venv\Lib\site-packages\numpy\_core\include -IC:\Users\maste\IdeaProjects\Minecraft-Overviewer\.venv\in
clude -IC:\Users\maste\AppData\Local\Programs\Python\Python313\include -IC:\Users\maste\AppData\Local\Programs\Python\Python313\Include "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.44.35207\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" /Tcoverviewer_core/src/Draw.c /Fobuild\temp.win-amd64-cpython-313\Release\overviewer_core\src\Draw.obj
Draw.c
C:\Users\maste\AppData\Local\Programs\Python\Python313\include\pyconfig.h(59): fatal error C1083: Cannot open include file: 'io.h': No such file or directory`

Does anyone know where this io.h file is supposed to come from?

**Comment by greatmastermario** [comment]
Figured it out. For reference, you need either the Windows 10 or Windows 11 SDK (depending on your OS) to prevent the issue. See https://stackoverflow.com/questions/40018405/cannot-open-include-file-io-h-no-such-file-or-directory

I should have a PR soon.

**Comment by greatmastermario** [comment]
Good news: Runtime errors are gone and the render completes.
Bad news: There are no textures in the renders. Need to spend more time on this when it's not almost midnight. If someone can spot the issue by checking out my branch, that will speed this along. 

Note that I am using the following environment and deps:
- Windows 11
- Building with VS
- setuptools 80.9.0
- numpy 2.3.2
- pillow 11.3.0
- networkx 3.5

There may have been some other API spec which changed, causing the render to behave unexpectedly.

**Comment by greatmastermario** [comment]
Last comment before bed: This is what my build directory looks like. I tried copying just the pyd file to my normal directory, but I think something else might be missing.

<img width="424" height="1086" alt="Image" src="https://github.com/user-attachments/assets/38ac39c0-451c-4b64-91d0-c5d6730501b6" />

**Comment by greatmastermario** [comment]
I have found out the rendering issue is (at least partially) related to other changes made in later Minecraft versions, including adding the "minecraft:" prefix to the chunk status and removing the world spawn from level.dat. I have fixes ready, but also ran into an issue with negative block states (see https://github.com/overviewer/Minecraft-Overviewer/issues/2113).
